.travis.yml

```diff
@@ -6,12 +9,7 @@ rvm:
 env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
 
 before_script:
-  - psql -c 'create database timeoverflow_test;' -U postgres
-  - RAILS_ENV=test bundle exec rake db:migrate
-
-sudo: false
-
-cache: bundler
+  - bundle exec rake db:setup db:migrate
```

Comment:
Not really, but.. could we forget to run migrations or add changes to
schema.rb?
I think it could be useful.. but not mandatory.
On 22 Mar 2016 8:39 pm, "Enrico Stano" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/coopdevs/timeoverflow/pull/191#discussion_r57054602>:
>
> > @@ -6,12 +9,7 @@ rvm:
> >  env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
> >
> >  before_script:
> > -  - psql -c 'create database timeoverflow_test;' -U postgres
> > -  - RAILS_ENV=test bundle exec rake db:migrate
> > -
> > -sudo: false
> > -
> > -cache: bundler
> > +  - bundle exec rake db:setup db:migrate
>
> The rails db:setup task will create the database, load the schema and
> initialize it with the seed data
>
> do we really need db:migrate here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coopdevs/timeoverflow/pull/191/files/8249b2b56da19f4e66107eb4fee4e13e34c2c360#r57054602>
>